pkgs/development/libraries/libsciter/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, glib
+, cairo
+, libuuid
+, pango
+, gdk-pixbuf
+, gtk3
+, stdenv
+, fetchurl
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "libsciter";
+
+  version = "4.4.8.23-bis";
```

Using `name` is depricated, you should only use `pname` and `version`:
```suggestion
  pname = "libsciter";
  version = "4.4.8.23-bis";
```